net-voip/ring-daemon/ring-daemon-99999999.ebuild

```diff
@@ -125,13 +126,18 @@ src_configure() {
 		rm -r src/pjproject
 	fi
 
+	#if system library is installed, then bundled will be ignored even with !system-
```

Comment:
Yes I also saw this problem with other libraries happening, but I don't know if it is intentional or a bug in the build system. But it can definitely be confusing.